tests/unittests/BackendTestUtils.cpp

```diff
@@ -33,7 +33,7 @@
 
 namespace glow {
 
-llvm::cl::OptionCategory operatorTestCat("OperatorTest Category");
```

Comment:
This was just a previously existing typo that you're fixing?

Reply:
Yep, think it was a copy paste typo.